### PR TITLE
Update POA information for claims reps

### PR DESCRIPTION
**Pull request by hinzed1127**
Updated the individual POA information, as well as some other cleanup/clarifications

## Related ticket
https://vajira.max.gov/browse/API-10869

### CODE REVIEW

File: test_accounts/benefits_test_accounts.md [hunk 1]
```diff
@@ -81,22 +83,33 @@ These test users have both Claims data and Health data and may be used in both t
...
+Test user 007 (John Doe) is a non-Veteran that can be added as a POA for any of the above Veterans (if not already). If test user 007 attempts to read or write data for a Veteran test user they do not have POA for, the server will return a `401 Unauthorized` error.
 
-Also, any of the above Veterans can also be Representatives for another Veteran. In addition to accessing the VA data for those the Veteran represents, he/she can also access his/her own personal VA data. If the Representative attempts to read or write data for a Veteran test user they do not have POA for, the server will return a 401 Unauthorized error.
+Also, any of the above Veterans can be a Representative for another Veteran. In addition to accessing the VA data for those the Veteran represents, they can also access their own personal VA data. If the Representative attempts to read or write data for a Veteran test user they do not have POA for, the server will return a `401 Unauthorized` error.
```
Comment (jw81) [inline test_accounts/benefits_test_accounts.md:94]: @SlakrHakr and I noticed this last week when we were dealing with some other POA stuff.  I'm pretty sure this line is incorrect.  

> "Also, any of the above Veterans can also be Representatives for another Veteran."

Aren't Tamara Ellis and John Doe the only people who can be representatives?

Comment (hinzed1127) [reply test_accounts/benefits_test_accounts.md:94]: Oh that's a good point. Pretty sure you're right. I'll just remove it.

File: test_accounts/benefits_test_accounts.md [hunk 2]
```diff
@@ -81,22 +83,31 @@ These test users have both Claims data and Health data and may be used in both t
...
 
 **Both Veterans and non-Veterans can be Representatives.**
 
-Test user 007 (John Doe) is a non-Veteran that can be added as a POA for any of the above Veterans (if not already). If test user 007 attempts to read or write data for a Veteran test user they do not have POA for, the server will return a 401 Unauthorized error.
-
-Also, any of the above Veterans can also be Representatives for another Veteran. In addition to accessing the VA data for those the Veteran represents, he/she can also access his/her own personal VA data. If the Representative attempts to read or write data for a Veteran test user they do not have POA for, the server will return a 401 Unauthorized error.
+Tamara Ellis, test user 001, is a Veteran. John Doe, test user 007, is not. Both can assigned POA for any of the above Veterans, if not already. If the Representative attempts to read or write data for a Veteran test user they do not have POA for, the server will return a `401 Unauthorized` error.
```
Comment (jw81) [inline test_accounts/benefits_test_accounts.md:92]: `Both can assigned POA` -> `Both can be assigned POA`

**Review by jw81 — CHANGES_REQUESTED** [review]
One VERY minor request.

File: test_accounts/benefits_test_accounts.md (hunk 2 above)
Comment (hinzed1127) [reply test_accounts/benefits_test_accounts.md]: [Done](https://github.com/department-of-veterans-affairs/vets-api-clients/pull/309/commits/66467bdcf6ab8caaa271e81bb13d37cc84000ac9)

**Pull request merged**